UndertaleModTool/CommunityScripts/FindObjectsWithSprite.csx

```diff
@@ -0,0 +1,87 @@
+﻿using System.Text;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UndertaleModLib.Util;
+using System.Text;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+EnsureDataLoaded();
+
+string[] spriteNames;
+ConcurrentBag<string> resultList = new();
+
+bool caseSensitive = ScriptQuestion("Case sensitive?");
+bool regexCheck = ScriptQuestion("Regex search?");
+string qText = regexCheck ? "Enter RegEx of sprite name" : "Enter sprite name(s)";
```

Comment:
```suggestion
string qText = regexCheck ? "Enter RegEx of sprite name(s)" : "Enter sprite name(s)";
```
For consistency?
Also am personally not a fan of very shortened names like this, would prefer `queryTitleText` or similar.